test/test_pooling.py

```diff
@@ -411,6 +411,41 @@ def find_one():
         # maxConnecting = unbounded: 30+ connections in ~0.140+ seconds
         print(len(pool.conns))
 
+    def test_csot_timeout_message(self):
+        with self.assertRaises(Exception) as error:
+            with timeout(0.001):
+                self.client.db.t.find_one({"$where": delay(1)})
+
+        self.assertTrue("(configured timeouts: timeoutMS: 1.0ms" in str(error.exception))
+
+    def test_socket_timeout_message(self):
+        client = rs_or_single_client(socketTimeoutMS=0.01)
```

Comment:
```
___________________ TestPooling.test_socket_timeout_message ____________________

self = <test.test_pooling.TestPooling testMethod=test_socket_timeout_message>

    def test_socket_timeout_message(self):
        client = rs_or_single_client(socketTimeoutMS=0.01)
    
>       with self.assertRaises(Exception) as error:
E       AssertionError: Exception not ### raised
```

Not sure how to explain the above error but could you increase the timeout to 500ms to make sure there's enough time to actually run the command (instead of timing out in server selection for example).